test/evergreen.yml

```diff
@@ -203,41 +203,49 @@ functions:
     - *configure_wiredtiger
     - command: shell.exec
       params:
-        working_dir: "wiredtiger/build_posix"
+        working_dir: "wiredtiger/cmake_build"
         shell: bash
         script: |
           set -o errexit
           set -o verbose
-          echo '#undef HAVE_FTRUNCATE' >> wiredtiger_config.h
+          echo '#undef HAVE_FTRUNCATE' >> config/wiredtiger_config.h
     - *make_wiredtiger
   "compile wiredtiger address sanitizer":
     - command: shell.exec
       params:
-        working_dir: "wiredtiger/build_posix"
+        working_dir: "wiredtiger"
         shell: bash
         script: |
           set -o errexit
           set -o verbose
-          sh reconf
           if [ "$OS" != "Windows_NT" ]; then
-            CC=/opt/mongodbtoolchain/v3/bin/clang CXX=/opt/mongodbtoolchain/v3/bin/clang++ PATH=/opt/mongodbtoolchain/v3/bin:$PATH CFLAGS="-ggdb -fPIC -fno-omit-frame-pointer -fsanitize=address" CXXFLAGS="-fsanitize=address -fno-omit-frame-pointer -ggdb -fPIC" \
-              ../configure ${configure_python_setting|} \
-              --enable-diagnostic --with-builtins=lz4,snappy,zlib,zstd
+            . test/evergreen/find_cmake.sh
+            if [ -d cmake_build ]; then rm -r cmake_build; fi
+            mkdir -p cmake_build
+            cd cmake_build
+            $CMAKE -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/mongodbtoolchain_v3_clang.cmake -DCMAKE_C_FLAGS="-ggdb" -DWITH_PIC=1 \
+              -DHAVE_DIAGNOSTIC=1 -DCMAKE_BUILD_TYPE=ASan \
+              -DHAVE_BUILTIN_EXTENSION_LZ4=1 -DHAVE_BUILTIN_EXTENSION_SNAPPY=1 -DHAVE_BUILTIN_EXTENSION_ZLIB=1 ${configure_python_setting|} \
+              -G "${cmake_generator|Ninja}" ../.
```

Comment:
I'm wonder how `-fno-omit-frame-pointer` is translated into the CMake ASAN build mode. It looks the `no_omit_frame_flag` is not set for ASAN build mode in `cmake/configs/modes.cmake` file.

Reply:
Good catch, `-fno-omit-frame-pointer` should be used in the CMake ASAN build mode (I completely missed this when refactoring the build modes). I've updated it here: cec889dbbfa2009222c0bd5729199ee4af86878c